### ISSUE TITLE
Account Tags searching options

### ISSUE THREAD

**Issue by basil-twisleton** [nuxsmin/sysPass#377]
I have been checking the latest beta of 2.0 hosted at https://demo-dev.syspass.org/
and noticed that when searching by Tags it only supports **OR.** 

It would be great it there was an option to use **AND** as well  or some additional logic based on text.

 eg
`+ is AND`
`- is AND NOT`
`| is OR`
`( ) are GROUPING`
`("web"+"sql"-"oracle") | ("web"+"sql"-"postgres")`
would show accounts with tags
(web **AND** sql without oracle) **OR** (web **AND** sql without postgres)

If that is too much work then a simple toggle to say search with **AND** instead of **OR.**

**Comment by basil-twisleton** [comment]
I have finally found a very useful scenario for this request. 

We have a number of servers that are no longer in production, however for compliance we keep offsite backups of the machines for a number of years. We do not want to remove the passwords as they may be needed if we ever have to restore from the backups.

I have implemented a tag to indicate that the account is no longer used in production. Being able to filter our these easily would be great. 

Or is there a better approach to do this?
